src/js/view/view.js

```diff
@@ -340,6 +340,45 @@ function View(_api, _model) {
         playerViewModel.change('playlistItem', onPlaylistItem);
         // Triggering 'resize' resulting in player 'ready'
         _lastWidth = _lastHeight = null;
+
+        // Setup floating scroll handler
+        const topOffset = 62;
+        if (_floatingConfig && _api.getEnvironment().OS.mobile) {
+            let canFire = true;
+            let debounceTO;
+
+            const checkFloat = () => {
+                const floating = _model.get('isFloating');
+                const hasCrossedThreshold = playerBounds.top < topOffset ?
+                    playerBounds.top <= window.scrollY :
+                    playerBounds.top <= window.scrollY + topOffset;
+
+                if (!floating && hasCrossedThreshold) {
+                    _updateFloating(0);
+                } else if (floating && !hasCrossedThreshold) {
+                    _updateFloating(1);
+                }
+            };
+
+            const throttledScroll = () => {
+                clearTimeout(debounceTO);
+                debounceTO = setTimeout(checkFloat, 150);
+
+                if (!canFire) {
+                    return;
+                }
+
+                canFire = false;
+                checkFloat();
+
+                setTimeout(() => {
+                    canFire = true;
+                }, 50);
+            };
+
+            window.addEventListener('scroll', throttledScroll);
```

Comment:
Is this listener only ever added once? Where is it removed?

Global listeners like this should be handled once in views-manager.js and update all player views.